src/contrasts.jl

```diff
@@ -114,7 +114,7 @@ for the base level (which defaults to the first level).
 # Constructors
 
 ```julia
-ContrastsMatrix{C <: AbstractContrasts}(contrasts::C, levels::AbstractVector)
```

Comment:
I think the parameter should be kept (else it's a duplicate of the following line).

Reply:
"Kept" doesn't mean "remove the line". That parameter was legitimate AFAICT if you want to specify `C`.

Reply:
The line is not part of the code. It is part of the documentation for constructors. I verified if any of the constructors used `C` and none of them did. Since none of the constructors methods use `C` it seems misleading to document that method when in fact it is equivalent in all regards to the other.